<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClangCodeModel::Internal::ClangProjectSettingsWidget</class>
 <widget class="QWidget" name="ClangCodeModel::Internal::ClangProjectSettingsWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>330</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="clangSettings">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Global</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="delayedTemplateParse">
     <property name="toolTip">
      <string>Parse templates in a MSVC-compliant way. This helps to parse headers for example from Active Template Library (ATL) or Windows Runtime Library (WRL).
However, using the relaxed and extended rules means also that no highlighting/completion can be provided within template functions.</string>
     </property>
     <property name="text">
      <string>Enable MSVC-compliant template parsing</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CppTools::ClangDiagnosticConfigsSelectionWidget" name="clangDiagnosticConfigsSelectionWidget" native="true"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CppTools::ClangDiagnosticConfigsSelectionWidget</class>
   <extends>QWidget</extends>
   <header>cpptools/clangdiagnosticconfigsselectionwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
